### DISCUSSION TITLE
Target line

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1524] [Q&A] [answered]
Hi all! I have 2 questions on target line.

1) for Ukraine, some indicators have target not for 2030, but for 2020. So I need to change this code for 2020 year:  

label_content: indicator.annotation_2030_target. If I just change the number it appears in code format on the graph. Where should I "translate" this code to the phrase?

2) on some graphs (e.g. https://ukraine-sdg.github.io/site/uk/15-3-4/, https://ukraine-sdg.github.io/site/uk/5-4-1/) target line lies on the up bottom and text is unreadable. How to add some place to the graph to keep the text visible?

Thank you for the help in advance!

**Comment by LucyGwilliamAdmin** [comment, marked as answer]
@ukraine-sdg

1. You can add the code to a new file called indicator.yml in your translation folders, and then add the field i.e. add `annotation_2020_target: 2020 indicator target` in translations/en/indicator.yml

2. To expand the graph you can the [graph_limits indicator config option](https://open-sdg.readthedocs.io/en/latest/indicator-configuration/#graph_limits)
